### DISCUSSION TITLE
Discrepancy Between Master Branch Code and NuGet Package

### DISCUSSION THREAD

**Discussion by moeller-projects** [stephanstapel/ZUGFeRD-csharp#447] [General]
While using the NuGet package, I noticed some differences between the package implementation and the code in the master branch of this repository. Specifically, the implementation in the NuGet package does not seem to reflect the latest updates or fixes present in the repository.

Could you kindly clarify when you plan to deploy a new version of the NuGet package that aligns with the current master branch?
Thank you for your time and support in maintaining this project. Looking forward to your response!

**Comment by DaveHalma** [comment]
Did you update your NuGet to 15.0.0 already?

**Comment by stephanstapel** [comment]
> While using the NuGet package, I noticed some differences between the package implementation and the code in the master branch of this repository. Specifically, the implementation in the NuGet package does not seem to reflect the latest updates or fixes present in the repository.
> 
> Could you kindly clarify when you plan to deploy a new version of the NuGet package that aligns with the current master branch? Thank you for your time and support in maintaining this project. Looking forward to your response!

There are plenty of things going on in the component currently that are worth being a new major version. There is no fine-grained branching and release process in the component so you will just have to wait a bit or use the github version until then which isn't a big deal to integrate into projects.
